src/makeRobot.js

```diff
@@ -38,7 +38,57 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
No need to duplicate these, you can rewrite like this
```suggestion
    name,
    wheels,
    version,
```

Reply:
when we pass the value this code will be like this ? :
  { name: 'robot1',
    version: 2,
    .............
}